BijlageA.md

```diff
@@ -1,37 +1,37 @@
-## Bijlage A: CRS overzicht tabel
+## Bijlage A: CRS-overzichttabel
 
-Onderstaande tabel geeft de naam en EPSG-code van veelgebruikte CRS-en in Nederland. Een beschrijving van de toepassingen van de CRS-en is te vinden op de [website van NSGI](https://www.nsgi.nl/coordinatenstelsels-en-transformaties/overzicht-coordinatenstelsels). Voor een lijst van CRS-en te gebruiken binnen INSPIRE wordt verwezen naar de [lijst gepubliceerd op de INSPIRE pagina's](https://inspire.ec.europa.eu/crs). Een ensemble CRS groepeert volgens EPSG en ISO 19111:2019 individuele realisaties van CRS-en en kan worden gebruikt waneer voor de beoogde toepassing en nauwkeurigheid het verschil tussen de individuele relaties niet significant is. Voor het WGS 84 ensemble zijn de verschillen tussen realisaties circa 2 meter, voor ETRS89 realisaties zijn de verschillen minder dan 0,1 meter.
+Onderstaande tabel geeft de naam en EPSG-code van veelgebruikte CRS-en in Nederland. Een beschrijving van de toepassingen van de CRS-en is te vinden op de [website van NSGI](https://www.nsgi.nl/coördinatenstelsels-en-transformaties/overzicht-coördinatenstelsels). Voor een lijst van CRS-en te gebruiken binnen INSPIRE wordt verwezen naar de [lijst gepubliceerd op de INSPIRE-pagina's](https://inspire.ec.europa.eu/crs). Een ensemble CRS groepeert volgens EPSG en ISO 19111:2019 individuele realisaties van CRS-en en kan worden gebruikt wanneer voor de beoogde toepassing en nauwkeurigheid het verschil tussen de individuele relaties niet significant is. Voor het WGS 84-ensemble zijn de verschillen tussen realisaties circa 2 meter, voor ETRS89-realisaties zijn de verschillen minder dan 0,1 meter.
 
```

Comment:
Het woord "coördinatenstelsels" is ook gecorrigeerd in de URL `https://www.nsgi.nl/coördinatenstelsels-en-transformaties/overzicht-coördinatenstelsels` maar die werkt daardoor niet meer.